### ISSUE TITLE
Agent on Alpine - False Trojaned version of file detected /bin/ls and others

### ISSUE THREAD

**Issue by Loic-Hakisa** [wazuh/wazuh#15858]
On Wazuh agent 4.3 installed on alpine docker image :
Rule: 510 fired (level 7) -> "Host-based anomaly detection event (rootcheck)."
Portion of the log(s):
Trojaned version of file '/bin/ls' detected. Signature used: 'bash|^/bin/sh|dev/[^clu]|\.tmp/lsfile|duarawkz|/prof|/security|file\.h' (Generic).
title: Trojaned version of file detected.
file: /bin/ls

Same problem for /usr/bin/env & /bin/echo.

How to reproduce : 
- `docker run -it --entrypoint sh php:8.1-fpm-alpine`
Then in container
- `strings /bin/echo | grep '/dev/[^n]'`

**Comment by jnasselle** [comment]
Hi @Loic-Hakisa ,

Alpine Linux uses Busybox instead of the widely used GNU coreutils. The main difference is that Busybox is a single binary/executable that contains all the common tools ([ref](https://linux.die.net/man/1/busybox)), rather than an executable per tool.
Rootcheck will search certain values in those executables based on the tool's purpose and will alert if an inappropriate usage was found.
In your case, Rootcheck will check `'bash|^/bin/sh|dev/[^clu]|.tmp/lsfile|duarawkz|/prof|/security|file.h'` against ALL Busybox tools, and will trigger false-positives since is checking the whole bundle and not only `/bin/ls`.

In conclusion: a new feature should be created to address Rootcheck validations against Busybox-like (bundled) binaries.

**Comment by jnasselle** [comment]
@Loic-Hakisa we will track this on https://github.com/wazuh/wazuh/issues/15947

**Comment by Loic-Hakisa** [comment]
Thanks @jnasselle for the excellent explanation !

**Comment by chemamartinez** [comment]
Solved at https://github.com/wazuh/wazuh/pull/15978 and https://github.com/wazuh/wazuh-packages/pull/2060
